### PR TITLE
Edit post

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Add edit feature of Post
user can edit the written post.
image

### CODE REVIEW

File: src/redux/slice/auth/authSlice.js [hunk 1]
```diff
@@ -27,7 +27,6 @@ export const authSlice = createSlice({
     [signupHandler.fulfilled]: (state, action) => {
       localStorage.setItem("token", action.payload.encodedToken);
       state.foundUser = action.payload.createdUser;
-      alert("signup success");
     },
     [signupHandler.rejected]: (action) => {
       console.log(action.payload);
```
Comment (anshtripathi01) [inline src/redux/slice/auth/authSlice.js:32]: Remove console.log

**Review by anshtripathi01 — APPROVED** [review]
-Remove console.log from everywhere.
-UI and Code looks good.

**Comment by Chiragmodi01** [comment]
Code looks good to me.
Few UI suggestions would be to add colors and borders, make the nav look separate, align like count and add margin left to profile card.

**Review by MonikaJhingan — APPROVED** [review]
Good to merge.

**Pull request merged**